fluent-bit-to-loki/pkg/lokiplugin/utils_test.go

```diff
@@ -459,6 +469,63 @@ var _ = Describe("Loki plugin utils", func() {
 			},
 		),
 	)
+	const (
```

Comment:
Use config.DefaultKubernetesMetadataTagKey and config.DefaultKubernetesMetadataTagPrefix instead of this

Reply:
the second one does not exists

Reply:
it exists